command.py

```diff
@@ -0,0 +1,244 @@
+"""
+Implementing CalicoBot commands
+"""
+import datetime
+import time
+import discord
+from discord import app_commands
+
+import embeds
+
+BOT = None
+def command_bot_init(main_bot):
+    """
+    A function to initialize command module
+    Args:
+        main_bot (discord.bot): set your bot instance to get channels
+    """
+    global BOT# pylint: disable=global-statement
+    BOT = main_bot
+
+user_karma_list=[]
+def command_init(main_list):
+    """A function to initialize command module
+    Args:
+        main_list (list): input your user karma data
+    """
+    global user_karma_list# pylint: disable=global-statement
+    user_karma_list = main_list
+
+class MemberCommands(app_commands.Group):
+    """MemberCommands Class
+
+        Its function will be registered when bot will start
+    """
+
+    def __init__(self, name: str):
+        super().__init__(name=name)
+    @app_commands.command(description="サーバーの情報を表示します")
+    async def serverinfo(self, interaction: discord.Interaction):
+        """
+        A command to show server infomation
+        """
+        guild=interaction.guild
+        guild_online_member=0
+        for online_member in guild.members:
+            if online_member.status == discord.Status.online:
+                guild_online_member =+ 1
+
+        role_list=[]
+        for role in guild.roles:
+            role_list.append(role.mention)
+        rlb=" | ".join(role_list)
+        embed=discord.Embed(title=f"{guild.name} - Info",color=discord.Color.blue(),
+                            timestamp=datetime.datetime.now())
+        embed.add_field(name="作成日",value=f"<t:{int(time.mktime(guild.created_at.timetuple()))}:R>")
+        embed.add_field(name="サーバーID",value=f"{guild.id}")
+        embed.add_field(name="所有者",value=guild.owner.mention)
+        embed.add_field(name=f"メンバー数({guild.member_count})",value=f"**{str(guild_online_member)}** オンライン\n**{guild.premium_subscription_count}** ブースター")
+        embed.add_field(name=f"チャンネル数({len(guild.channels)})",value=f"**{len(guild.text_channels)}** テキストチャンネル\n**{len(guild.voice_channels)}** ボイスチャンネル")
```

Comment:
These lines seem too long.
```suggestion
                embed.add_field(name=f"メンバー数({guild.member_count})",
                        value="\n".join([f"**{str(guild_online_member)}** オンライン",
                                         f"**{guild.premium_subscription_count}** ブースター"])
                        )
        embed.add_field(name=f"チャンネル数({len(guild.channels)})",
                        value="\n".join([f"**{len(guild.text_channels)}** テキストチャンネル",
                                         f"**{len(guild.voice_channels)}** ボイスチャンネル"])
                        )
```